.gitignore

```diff
@@ -1,6 +1,11 @@
-__pycache__/
 .env
+__pycache__/
 data/
 backend/src/source_list.json
-.ruff_cache
-backend/src/secret.json
\ No newline at end of file
+backend/src/secret.json
+*.pyc
+frontend/*.json  
+evaluation/human_evaluation/*.json  
+*.json
```

Comment:
Are you sure? This will lead to all jsons being ignored.

Reply:
Yes `/*.json` is the way to ignore all root dir json files. The way `*.json` works is by globbing, so it matches everything that ends with `.json`. 

https://github.com/kenmueller/gitignore